assume_role.sh

```diff
@@ -58,8 +59,14 @@ if [ -n "$destinationAccountNumber" ] && [ -n "$sourceAccountNumber" ] && [ -n "
     export AWS_SESSION_TOKEN=$arg2
     arg3=$(echo "$commandResult1" | cut -d "," -f3)
     export AWS_ACCESS_KEY_ID=$arg3
-  fi
+  else
+    echo "Unable to assume role"
+    exitCode=1
+  if
+
+  (exit $exitCode)
```

Comment:
move to end and comment indicating that parens protect bash session from destruction